apps/examples/slsiwifi/slsiwifi_main.c

```diff
@@ -104,7 +105,7 @@ static struct mm_heap_s *g_user_heap;
 int g_memstat_total = 0;
 static int getMemUsage(void)
 {
-	g_user_heap = mm_get_heap_info();
+	g_user_heap = mm_get_heap_info_with_index(BASE_HEAP_INDEX);
```

Comment:
In this case, how one can be sure that only base index heap is being used?

Reply:
This sample is tested based on single heap. So base heap is used.